pandas/core/arrays/base.py

```diff
@@ -640,8 +637,8 @@ def factorize(self, na_sentinel=-1):
 
     @Substitution(klass='ExtensionArray')
     @Appender(_extension_array_shared_docs['repeat'])
-    def repeat(self, repeats, *args, **kwargs):
-        nv.validate_repeat(args, kwargs)
+    def repeat(self, repeats, axis=None):
+        nv.validate_repeat(tuple(), dict(axis=axis))
```

Comment:
I think these could be written more explicitly as:
```python
if axis is not None:
    raise ValueError('...')
```
I don't think `nv.validate_repeat` is doing more than that but could be missing something.  The advantage to using `nv.validate_repeat` simply being that the error message is kept in a single place instead of being explicitly written in each method.

Reply:
this is exactly why we have it this way